documentation/builders/components/power/batterymonitor.md

```diff
@@ -31,3 +31,60 @@ The script in [src/jukebox/components/battery_monitor/batt_mon_i2c_ads1015/\_\_i
 >
 > * the circuit is constantly draining the battery! (leak current up to: 2.1µA)
 > * the time between sample needs to be a minimum 1sec with this high impedance voltage divider don't use the continuous conversion method!
+
+# Battery Monitor based on an INA219
+
+> [!CAUTION]
```

Comment:
Is the caution part special to the INA or is it generic to all battery monitors?

Reply:
Made it generic